types/pi-camera/pi-camera-tests.ts

```diff
@@ -0,0 +1,41 @@
+import PiCamera from 'pi-camera';
```

Comment:
`import PiCamera = require('pi-camera')` is the import that matches the export above.